vclib-openid/src/commonTest/kotlin/at/asitplus/wallet/lib/oidvci/OidvciProcessTest.kt

```diff
@@ -94,7 +96,9 @@ class OidvciProcessTest : FunSpec({
         val jws = JwsSigned.parse(serializedCredential.substringBeforeLast("~"))
         jws.shouldNotBeNull()
         val sdJwt = VerifiableCredentialSdJwt.deserialize(jws.payload.decodeToString())
-        sdJwt.shouldNotBeNull().also { println(it) }
+            .getOrThrow().shouldNotBeNull()
+
+        println(sdJwt)
```

Comment:
Is `println` better than using `Napier.d` here?

Reply:
In tests it doesn't matter really, I think